pennylane_lightning/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.24.0-dev16"
+__version__ = "0.24.0-rc0"
```

Comment:
```suggestion
__version__ = "0.24.0-dev17"
```

Reply:
Please double-check if it would be dev17.

Reply:
Thanks @AmintorDusko ! Need confirmation from @maliasadi and @chaeyeunpark about the version.

Reply:
I think we need to merge this after releasing 0.24.0. Then the version should start with `0.25.0-dev`. Let's just wait until we have a version bump.

Reply:
Thanks @chaeyeunpark !